pkg/cmd/admin/policy/reconcile_clusterrolebindings.go

```diff
@@ -193,6 +216,11 @@ func (o *ReconcileClusterRoleBindingsOptions) ChangedClusterRoleBindings() ([]*a
 		}
 	}
 
+	if len(rolesNotFound) != 0 {
+		// return the known changes and the error so that a caller can decide if he wants a partial update
+		return changedRoleBindings, fmt.Errorf("did not find requested cluster role %s", rolesNotFound.List())
```

Comment:
You need to update callers (`RunReconcileClusterRoleBindings` comes to mind), to handle partial success.